locale/en/blog/uncategorized/10-lts-to-12-lts.md

```diff
@@ -0,0 +1,4333 @@
+* **http**:
+  * switch default parser to llhttp (Anna Henningsen) [#24870](https://github.com/nodejs/node/pull/24870)
+* **os**:
+  * implement `os.type()` using `uv_os_uname()` (cjihrig) [#25659](https://github.com/nodejs/node/pull/25659)
+* **policy**:
+  * Experimental support for module integrity checks through a manifest file is implemented now. [nodejs/node#23834](https://github.com/nodejs/node/pull/23834)
+  * Added `--policy-integrity=sri` CLI option to mitigate policy tampering. If a policy integrity is specified and the policy does not have that integrity, Node.js will error prior to running any code (Bradley Farias) [#28734](https://github.com/nodejs/node/pull/28734).
+* **process**:
+  * Add source-map support to stack traces by using `--enable-source-maps`[#29564](https://github.com/nodejs/node/pull/29564)
+  * A new method, `process.resourceUsage()` was added. It returns resource usage for the current process, such as CPU time [#28018](https://github.com/nodejs/node/pull/28018).
+* **report**:
+  * An experimental diagnostic API for capturing process state is available as `process.report` and through command line flags. [nodejs/node#22712](https://github.com/nodejs/node/pull/22712)
+  * The cpu info got added to the report output [#28188](https://github.com/nodejs/node/pull/28188)
+* **repl**:
+  * The REPL now supports multi-line statements using `BigInt` literals as well
+    as public and private class fields and methods (Ruben Bridgewater) [#27400](https://github.com/nodejs/node/pull/27400).
+  * The REPL now supports tab autocompletion of file paths with `fs` methods (Anto Aravinth) [#26648](https://github.com/nodejs/node/pull/26648).
+* **tls**:
+  * support TLSv1.3 (Sam Roberts) [#26209](https://github.com/nodejs/node/pull/26209)
+  * disable TLS v1.0 and v1.1 by default (Ben Noordhuis) [#23814](https://github.com/nodejs/node/pull/23814)
+* **util**:
+  * The WHATWG `TextEncoder` and `TextDecoder` are now globals. [#22281](https://github.com/nodejs/node/pull/22281)
+* **v8**
+  * Added `v8.getHeapSnapshot` and `v8.writeHeapSnapshot` to generate snapshots in the format used by tools such as Chrome DevTools (James M Snell) [#26501](https://github.com/nodejs/node/pull/26501).
+* **worker**:
+  * The `worker_thread` module is now stable [#29512](https://github.com/nodejs/node/pull/29512)
+
+Addtionally, addtions to collaborators and TSC:
+
+* **meta**:
+
+  * Added [Jiawen Geng](https://github.com/gengjiawen) to collaborators [#28322](https://github.com/nodejs/node/pull/28322).
+  * Added [Christian Clauss](https://github.com/cclauss) to collaborators [#27554](https://github.com/nodejs/node/pull/27554).
+  * Added [Yongsheng Zhang](https://github.com/ZYSzys) to collaborators [#26730](https://github.com/nodejs/node/pull/26730).
+  * [Gireesh Punathil](https://github.com/gireeshpunathil) is now a member of the Technical Steering Committee [#26657](https://github.com/nodejs/node/pull/26657).
```

Comment:
These are notable but I wonder if it's more things that happened in the time period rather than being specific to the version of Node.js (they're in the release notes because the README gets updated).

Reply:
It is indeed things that happened within the time period between 10 LTS and 12 LTS. I still think it's valuable to include in this context :)